src/main/java/teammates/ui/controller/AdminEmailTrashAction.java

```diff
@@ -17,7 +17,7 @@ protected ActionResult execute() {
         String emailId = getRequestParamValue(Const.ParamsNames.ADMIN_EMAIL_ID);
         
         String redirect = getRequestParamValue(Const.ParamsNames.ADMIN_EMAIL_TRASH_ACTION_REDIRECT);
-        redirect = redirect == null ? Const.ActionURIs.ADMIN_EMAIL_TRASH_PAGE : redirect;
+        redirect = redirect == null ? Const.ActionURIs.ADMIN_EMAIL_TRASH_PAGE : redirect; // NOPMD
```

Comment:
When this is about to be merged, I'll make an issue to remove these NOPMDs when we upgrade PMD version